### PR TITLE
Code review exercise

### PR DESCRIPTION
**Pull request by mariegrace31**
Practicing the code review on my own project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <p>Hello,world, How are you!</p>
+</body>
```
Comment (mariegrace31) [inline index.html:11]: Please, instead of putting your text in a _p_ tag, it would be better if you put it in an _h1_ tag, as it's an important heading.

**Review by mariegrace31 — COMMENTED** [review]
Good job so far but there are some issues you still need to work on.
**REQUIRED CHANGES**
-It would be better if you add a structured README file so that a person going through your project can understand what your project is about. You can use this template https://github.com/microverseinc/readme-template and apply changes according to your project.
**Optional suggestions**
-[OPTIONAL] Please, your commits messages must be more professional. Instead of saying "HTML" you can say "Add html file" and it would be better if your commits messages are capitalized, it gives your project a professional look.
Please check my comments below and make all required changes.
Submit for a re-review when you're done.
Thanks 😉

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @mariegrace31  👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 
✔ Great review👍 
✔ Good use of the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)👍 
## Required Changes ♻️

- [ ]  Kindly as suggested in the project requirements, It would be better to Add as many comments as possible in the given time and finish your review. Remember to add a minimum of 2 comments 👍 
## Optional request 
- [ ] **[Optional]** I think It would be better to use a couple of emojis or gifs in the review to make the code review more human-friendly 👽😍

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mariegrace31) [inline index.html:11]: It would be better to add a CSS file, to style your work and it would give it a better look.

**Review by mariegrace31 — COMMENTED** [review]
Good job so far. But there are some issues that you still need to work on.
Please check my comments below and make the required changes.
submit for a re-review when you're done.
Thanks😘😘😘

**Review by ABDULALI3468 — APPROVED** [review]
Hi @mariegrace31 , 👍 

# APPROVED 💯 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
☑️Used suggestions politely.
☑️Good use of the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)👍


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**